gce/burst-training/README.md

```diff
@@ -0,0 +1,242 @@
+# Compute Engine burst training
+
+[Google Compute Engine](https://cloud.google.com/compute/) gives you access to
+very powerful virtual machines which you can use for as much or, more
+importantly for most data science use cases, as little time as you need.
+
+Common tasks that might take tens of minutes or even hours on your personal can
+be completed in seconds on a machine with 64 cores and almost half a terabyte of
+memory, and this can be done at a cost of under $1.
```

Comment:
Is the under $1 dollar claim always going to be true?

Reply:
Nope, I will change this to "at a low cost".

Reply:
Actually, went back and read the paragraph. The "under $1" applies to the "10s of minutes/hours -> seconds" compression. How would we clarify that message?

Reply:
Changed that paragraph, please take a look @puneith